SeaPublicWebsite/Helpers/Global.cs

```diff
@@ -0,0 +1,18 @@
+﻿using Microsoft.Extensions.Options;
+
+namespace SeaPublicWebsite.Helpers;
+
+public class GlobalConstants
+{
+    public readonly GlobalConstantsConfiguration Configuration;
+    public GlobalConstants(IOptions<GlobalConstantsConfiguration> options)
+    {
+        Configuration = options.Value;
+    }
+}
+
+public class GlobalConstantsConfiguration
+{
+    public const string ConfigSection = "GlobalConstants";
+    public string ServiceName { get; set; }
+}
```

Comment:
Let me know of you think of a better way to add a 'global' type of configuration

Reply:
I think that, given that this will be for things that are actually constant, we can go much simpler and just a have a `Constants.cs` file in the root of the project along the lines of:

```
public static class Constants
{
    public const string SERVICE_NAME = "Improve your property's energy efficiency"
}
```